db/migrate/20241104135353_comment_doorkeeper_tables.rb

```diff
@@ -0,0 +1,53 @@
+class CommentDoorkeeperTables < ActiveRecord::Migration[7.1]
+  def change
+    change_column_comment(:oauth_applications, :uid, from: nil, to: <<~COMMENT
```

Comment:
```suggestion
    change_column_comment(:oauth_applications, :uid, from: nil, to: <<~COMMENT.strip
```

le `.strip` permet d’éviter le \n qui traîne à la fin de ces commentaires. 

Et très cools ces commentaires sur les colonnes de la db, j’allais dire que j’avais un doute je n’avais pas vu qu’ils apparaissent aussi dans le schema.rb ce qui est super 👍

Reply:
merci pour la suggestion du strip ! Et en plus les commentaires apparaissent dans metabase (au hover sur les noms des colonnes), donc c'est utile aussi pour l'équipe non-tech

Reply:
Il faut re-générer `schema.rb` après l'ajout de ce strip. :wink: 

Et sinon, je me pose la question de la cohérence : tous les commentaires actuels finissent pas `\n`, est-ce que c'est si grave si on continue ?

Et dernière remarque : si on ajoute le `strip` à cette ligne, autant ajouter des `strip` aux autres lignes de ce fichier de migration.